providers/hubspot/webhook.go

```diff
@@ -0,0 +1,47 @@
+package hubspot
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/amp-labs/connectors/common"
+)
+
+type WebhookMessage struct {
+	AppId            int    `json:"appId"`
+	EventId          int    `json:"eventId"`
+	SubscriptionId   int    `json:"subscriptionId"`
+	PortalId         int    `json:"portalId"`
+	OccurredAt       int    `json:"occurredAt"`
+	SubscriptionType string `json:"subscriptionType"`
+	AttemptNumber    int    `json:"attemptNumber"`
+	ObjectId         int    `json:"objectId"`
+	ChangeSource     string `json:"changeSource"`
+	PropertyName     string `json:"propertyName"`
+	PropertyValue    string `json:"propertyValue"`
+}
+
+type WebhookResult struct {
```

Comment:
What's the rationale for introducing a WebhookResult type? It seems a bit unusual that the function below simply puts the `msg` parameter inside of `WebhookMessage`, since the caller of this method will already have `msg`. It would be simpler if the function just returned `*common.ReadResultRow`. Also `WebhookMessage` will look different for each provider, whereas `common.ReadResultRow` will look the same across providers, so by returning only `*common.ReadResultRow`, we can more easily make this method into an interface later when we support subscribe actions for other providers.

Also the function name is too verbose, maybe something like `GetRecordFromWebhook`?

So then the signature becomes:

```
func (c *Connector) GetRecordFromWebhook(
  ctx context.Context, msg *WebhookMessage,
 )(*common.ReadResultRow, error)
```